### ISSUE TITLE
currentStatusBarHeight not working

### ISSUE THREAD

**Issue by npomfret** [brentvatne/react-native-status-bar-size#13]
Works initially, but after a while it seems to say 20 when its zero, and zero when its 20.

**Comment by mikesurowiec** [comment]
~~I'm also experiencing the same issue. The initial height is correct, but after hiding / showing the status bar, the event listeners never seem to fire~~

On master it is mostly working for me, it just doesn't fire the listeners when `StatusBar.setHidden()` is called

**Comment by nonotest** [comment]
yes it looks like `willchange` is correct and `didChange` is wrong, shows the old value.

**Comment by connor-ricks** [comment]
Agreed, not quite sure why this is.
